pages/windows/cl.md

```diff
@@ -0,0 +1,32 @@
+# cl
+
+> The Microsoft C/C++ compiler for compiling and linking source code files.
+> More information: <https://learn.microsoft.com/cpp/build/reference/compiler-command-line-syntax>.
+
+- Compile a source file:
+
+`cl {{path/to/source.c}}`
+
+- Compile and create an executable with a custom name:
+
+`cl /Fe{{path/to/output_executable}} {{path/to/source.c}}`
```

Comment:
Does this need to be next to the path or can there be a space in between /Fe and the path?